### PR TITLE
Add coq-belgames.1.1.0 in released

### PR DESCRIPTION
**Pull request by erikmd**
(Cc @pPomCo pour info)

### CODE REVIEW

File: released/packages/coq-belgames/coq-belgames.1.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  "coq" {(>= "8.14" & < "8.17~")}
+  "coq-mathcomp-algebra" {(>= "1.13" & < "1.16~")}
+]
+
+tags: [
+  "keyword:games of incomplete information"
+  "keyword:belief functions theory"
+  "keyword:mathcomp/ssreflect"
+  "logpath:BelGames"
+]
```
Comment (silene) [inline released/packages/coq-belgames/coq-belgames.1.1.0/opam:34]:
```suggestion
  "date:2023-02-24"
]
```

File: released/packages/coq-belgames/coq-belgames.1.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+install: [make "install"]
+depends: [
+  "coq" {(>= "8.14" & < "8.17~")}
+  "coq-mathcomp-algebra" {(>= "1.13" & < "1.16~")}
+]
+
+tags: [
+  "keyword:games of incomplete information"
+  "keyword:belief functions theory"
+  "keyword:mathcomp/ssreflect"
```
Comment (silene) [inline released/packages/coq-belgames/coq-belgames.1.1.0/opam:32]:
```suggestion
  "keyword:mathematical components"
  "category:???"
```

No other package is using that keyword, but some of them are "`mathematical components`". Using it depends on whether you think your package is part of the Mathematical Components landscape (or just uses it).

Also having a category would be great. Not sure which one, however.

Comment (erikmd) [reply released/packages/coq-belgames/coq-belgames.1.1.0/opam:32]: Good point… better choose standard keywords/categories.
(We especially wanted to emphasize that we strongly rely on the mathcomp libraries so coq-belgames is compatible with them; given you suggest this is not enough to select this keyword, I'm going to remove it.)

Comment (erikmd) [reply released/packages/coq-belgames/coq-belgames.1.1.0/opam:32]: Should I also update the 1.0.0 keywords in this very same PR? or open another PR.

Comment (silene) [reply released/packages/coq-belgames/coq-belgames.1.1.0/opam]: As you wish. Both are fine with me.

**Comment by erikmd** [comment]
> Should I also update the keywords of coq-belgames.1.0.0 in this very same PR? or open another PR.

On second thought, I realize opening another PR may be better w.r.t. CI… will do it now.

**Pull request merged**